test/browser/commands.test.js

```diff
@@ -31,6 +31,38 @@ describe('commands', () => {
 			window.removeEventListener('click', onClick);
 		});
 
+		it('should click at top-left by default', async() => {
+			const clickPos = { x: 0, y: 0 };
```

Comment:
Since this and the `mousePos` stuff is now used in a bunch of tests, can we pull it out and set it up in a `before` just inside the `describe('click/hover')` block?

Reply:
Yup sounds good! Thanks for the rec!